src/realtime/commands/direct.commands.ts

```diff
@@ -199,6 +203,8 @@ export class DirectCommands {
                 node_type: 'item',
                 reaction_type: reactionType || 'like',
                 reaction_status: reactionStatus || 'created',
+                targetItemType,
```

Comment:
This should be snake-case.
```suggestion
                target_item_type: targetItemType,
```